testing/tests/DevExpress.ui.widgets/listParts/editingTests.js

```diff
@@ -408,6 +408,24 @@ QUnit.test('selectAll/unselectAll for \'allPages\' selectAllMode', function(asse
     assert.equal(loading.callCount, 2, 'no load during unselect all');
 });
 
+QUnit.test('unselectAll method should not unselect disabled items', function(assert) {
+    const items = [{ text: '1', disabled: true }, { text: '2' }];
+    const instance = $('#list').dxList({
+        dataSource: items,
+        selectedItemKeys: '1',
+        selectionMode: 'all',
+        keyExpr: 'text',
+    }).dxList('instance');
+
+    instance.selectAll();
+
+    assert.deepEqual(instance.option('selectedItems'), items, 'all item are selected');
```

Comment:
Why all items should be selected here? Shouldn't it be ignored as in the test below?

Reply:
`disabled` item is selected on init, so `selectAll()` here just selects second item (not disabled).
actually there is no need in this, I have refactored this test - now they are both selected on init and then we call `unselectAll()`